.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 before_script:
   - ./tools/run_ganache_cli.sh </dev/null 1>/dev/null 2>&1 &
 script:
-  - npm build
+  - npm run restart
```

Comment:
perhaps the script can be renamed, but I get the point i think

Reply:
Definitely agree. Was trying to find a better name, however, because of time constraints moved with that one. It's intention is:

Once the branch is changed, or repo is cloned, the script updates dev dependencies and initialize npm modules. Hence, maybe:

`npm run update` ... the only shortcoming is that there is `npm update`, however, we can maybe leave with that one :). Objections?

Reply:
I've pushed an update.